pkg/aws_config_client/completer.go

```diff
@@ -252,10 +292,17 @@ func (c *completer) Complete(out *ini.File) error {
 	if err != nil {
 		return err
 	}
+	if len(profiles) == 0 {
```

Comment:
how does this compare with L280?
shouldn't we essentially never reach this case? should this be an error?

Reply:
This case is different because it is a Survey() output! I'm deleting this block, not because it's similar to L280 but because c.assembleAWSConfig() handles empty profiles already.